iib/workers/tasks/build.py

```diff
@@ -134,9 +134,17 @@ def _finish_request_post_build(output_pull_spec, request_id, arches):
     :param set arches: the set of arches that were built as part of this request
     :raises IIBError: if the manifest list couldn't be created and pushed
     """
+    conf = get_worker_config()
```

Comment:
optional: We can move this to a separate helper method.

Reply:
This function is already quite small and the new code won't be used outside of this function, so I think it's okay as is.